labelbox/schema/ontology.py

```diff
@@ -127,6 +128,12 @@ class Type(Enum):
     schema_id: Optional[str] = None
     feature_schema_id: Optional[str] = None
 
+    def __post_init__(self):
+        if self.class_type == Classification.Type.DROPDOWN:
+            warnings.warn(
+            "Dropdown classification is deprecated and will be "
+            "removed in a future release")
```

Comment:
I think we want to also warn users that dropdown tools cannot be created in the editor after 3/31.